pkg/cli/testdata/MockClient.go

```diff
@@ -142,6 +143,7 @@ func (m *MockClient) AppGet(ctx context.Context, name string) (*apiv1.App, error
 	case "dne":
 		return nil, fmt.Errorf("error: app %s does not exist", name)
 	case "found":
+
```

Comment:
this is the only case that has a newline between the case and the logic. can you remove it for consistency?